README.md

```diff
@@ -42,12 +44,59 @@ to query the vector store and review potential SAST errors.
 ### Evaluation
 - Applies metrics (from Ragas library) to assess the quality of model outputs.
 
-## Evaluation & Metrics 📊
+## 📊 Evaluation & Metrics
 The evaluations of the model responses are being done using the following metrics:
 - **Response Relevancy:**  
   Ensures that the generated answers are directly related to the query.  
   [Response Relevancy](https://docs.ragas.io/en/latest/concepts/metrics/available_metrics/answer_relevance/).
   
 
-## Installation & Setup 🔌
+## 🔌 Installation & Setup 
 Please refer to [how to run](./docs/setup.md) guideline.
+
+
```

Comment:
Please move this entire section into setup.md file. This content is more appropriate there.